products/shopware.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Shopware
+category: server-app
+tags: php-runtime
+iconSlug: shopware
+permalink: /shopware
+versionCommand: ./bin/console -V
+releasePolicyLink: https://www.shopware.com/en/news/shopware-6-versioning-strategy/
+changelogTemplate: "https://www.shopware.com/en/changelog/#release-{{'__LATEST__' | replace:'.','-'}}"
+releaseLabel: "{{'__RELEASE_CYCLE__' | replace:'.','-'}}"
```

Comment:
https://developer.shopware.com/docs/ and https://github.com/shopware/shopware/releases/tag/v6.5.6.0 are using dots. Why using dashes for release label ?